package.json

```diff
@@ -66,6 +65,7 @@
     "marked": "^8.0.0",
     "path-browserify": "^1.0.1",
     "process": "^0.11.10",
+    "stylelint-use-logical-spec": "^5.0.0",
```

Comment:
Just realized that this was added as a Dependency and not a DevDependency

Reply:
Darn, you're right